### DISCUSSION TITLE
For fuseki, how do I download all RDF triples from a sparql endpoint (if RDF triples are a lot) ?

### DISCUSSION THREAD

**Discussion by chenkuncloud** [apache/jena#1536] [Q&A] [answered]
For fuseki, how do I download all RDF triples from a sparql endpoint (if RDF triples are a lot) ?

**Comment by afs** [comment, marked as answer]
If you do an HTTP GET request to the dataset URL, with Content-Type application/n-quads then the server will stream the N-Quads.

The configuration needs to have operation "gsp-r" or "gsp-rw" on the dataset URL - which is the case in the default configuration.

**Comment by chenkuncloud** [comment]
Thank you very much for your reply !

Best Regards
